cmake/AwsPrebuildDependency.cmake

```diff
@@ -75,3 +92,52 @@ function(aws_prebuild_dependency)
         DESTINATION ${CMAKE_INSTALL_PREFIX}
     )
 endfunction()
+
+# Get list of optional or platform-specific variables that may affect build process.
+function(aws_get_variables_for_prebuild_dependency AWS_CMAKE_PREBUILD_ARGS)
+    set(variables "")
+    set(variablesToIgnore CMAKE_INSTALL_PREFIX)
```

Comment:
trivial: I don't see `variablesToIgnore` being used anywhere. Should it be deleted, or did we forget to do a check later?

Reply:
Yep, it's a remaining of the previous attempts to make this more generic, removed it